### DISCUSSION TITLE
How to get an instance of an ILanguageConfiguration from outside of TM4E plugin?

### DISCUSSION THREAD

**Discussion by rubenporras** [eclipse/tm4e#473] [Q&A]
Hi,

I would appreciate help on how to get an instance of org.eclipse.tm4e.languageconfiguration.ILanguageConfiguration, so that I can call getComments() on that.

Currently, which the classes exported in the package org.eclipse.tm4e.languageconfiguration I do not see how to do that. Do we have some registry that one can access? If so, which class offers access to the registry?

Regards

**Comment by angelozerr** [comment]
Today all classes are internal and not exported. Innother words à refactoring must be done
